src/components/basic/Alert.md

```diff
@@ -0,0 +1,89 @@
+## Site Alerts
```

Comment:
This applies to all items in the md file. Do we want to provide examples for each or one example for them all, with an HTML Snippet as we have for the other components?

Reply:
yep, that was my bad, will update.

Reply:
added.

Reply:
HTML Snippets are present and look good.